SimTracker/TrackHistory/src/TrackHistory.cc

```diff
@@ -29,6 +29,15 @@ TrackHistory::TrackHistory(const edm::ParameterSet &config, edm::ConsumesCollect
   quality_ = 0.;
 }
 
+void TrackHistory::fillPSetDescription(edm::ParameterSetDescription &desc) {
+  desc.addUntracked<edm::InputTag>("trackProducer", edm::InputTag(""));
+  desc.addUntracked<edm::InputTag>("trackingTruth", edm::InputTag(""));
+  desc.addUntracked<edm::InputTag>("trackAssociator", edm::InputTag(""));
```

Comment:
are these left empty to avoid choosing a default between [TrackHistory_cff.py](https://github.com/cms-sw/cmssw/blob/a83766bfe6fb83b7868264fd00b19f5fbfeefbb7/SimTracker/TrackHistory/python/TrackHistory_cff.py) and [MuonHistory_cff.py](https://github.com/cms-sw/cmssw/blob/a83766bfe6fb83b7868264fd00b19f5fbfeefbb7/SimTracker/TrackHistory/python/MuonHistory_cff.py)? I would treat `TrackHistory_cff.py` as the default.

Reply:
I don't care much about the actual default, provided there is one (this is all confdb cares about).